server/controllers/interview.js

```diff
@@ -0,0 +1,111 @@
+// connect to the Interview Model
+const Interview = require("../models/interview");
+const User = require("../models/user");
+
+module.exports.createInterview = async (req, res, next) => {
+  const { guest, date, theme, questions, difficulty } = req.body;
+  const user = await User.findOne({ email: req.user.user }, "id");
+
+  const newInterview = new Interview({
+    owner: user,
```

Comment:
Here you should have the user._id coming from the auth middleware correct? If not that should be added there rather than calling findOne here

Reply:
The middleware is now sending the userId, it was achieved by modifying signin.js. I would appreciate a review of that file as I am not sure if I used best practices there.